src/state/actions/auth/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './setProfileId';
```

Comment:
Do we need to export *  if the functions and variables are exported inline?

Reply:
My intention was to make the action constants, i.e. `SET_USER_PROFILE` easily accessible. For instance `import { SET_USER_PROFILE } from './src/state/actions';` as opposed to `import { SET_USER_PROFILE } from './src/state/actions/userProfile/setUserProfile'`. However, this does add like 6 files to the codebase and might cause some confusion if anyone is trying to find where an action constant is coming from. I'll go ahead and simplify this approach.